docs/release_notes/0.4.0.md

```diff
@@ -1,10 +1,24 @@
 # Release 0.4.0
 
 ## Features
-- ...
+
+Includes all features of the previous version, plus:
+- New experimental feature `generate profile` (#1132)
+- Added installation of Helm/Tiller as part of the `install flux` operation (#1136)
```

Comment:
Let's use full command names, it's not clear otherwise:
```suggestion
- Added installation of Helm/Tiller as part of the `eksctl install flux` command (#1136)
```